content/blog/2022-06-01-Jupyter-in-cloud-with-TPI.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Jupyter Notebooks in any cloud with TPI
+date: 2022-06-01
+description: >
+  Use TPI to quickly and cheaply provision a cloud-hosted Jupyter server to run
+  your notebooks on.
+descriptionLong: >
+  We can use TPI to quickly and cheaply provision cloud instances. This guide
+  will explore how to use TPI to launch a Jupyter server on an Amazon Web
+  Services EC2 instance at the lowest possible cost. With minor modifications,
+  you can also launch instances on Google Cloud Platform, Microsoft Azure, or
+  Kubernetes.
```

Comment:
For the description, I think it would be good to include @casperdcl 's "bring your own cloud" catchphrase.  So after the first sentence, maybe:
This bring-your-own-cloud approach can launch instances on Google Cloud, Microsoft Azure, or Kubernetes with minor modifications.